src/tools/info/sys_info.c

```diff
@@ -30,9 +30,9 @@ static const char* cpu_model_names[] = {
     [UCS_CPU_MODEL_ARM_AARCH64]       = "ARM 64-bit",
     [UCS_CPU_MODEL_AMD_NAPLES]        = "Naples",
     [UCS_CPU_MODEL_AMD_ROME]          = "Rome",
-    [UCS_CPU_MODEL_ZHAOXIN_Zhangjiang]= "Zhangjiang",
-    [UCS_CPU_MODEL_ZHAOXIN_Wudaokou]  = "Wudaokou",
-    [UCS_CPU_MODEL_ZHAOXIN_Lujiazui]  = "Lujiazui"
+    [UCS_CPU_MODEL_ZHAOXIN_ZHANGJIANG] = "Zhangjiang",
+    [UCS_CPU_MODEL_ZHAOXIN_WUDAOKOU]  = "Wudaokou",
+    [UCS_CPU_MODEL_ZHAOXIN_LUJIAZUI]  = "Lujiazui"
```

Comment:
pls indent all other entries as well, so the '=' will be con same column

Reply:
fixed

Reply:
@yosefe i only add space, why did it report an error?

Reply:
error is unrelated, I just restarted failed jobs

Reply:
@brminich Still reporting an error

Reply:
@brminich Still reporting an error, Is it a problem with the test script?

Reply:
unrelated issue, which should be fixed by #5949, let's restart testing when it is merged